src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -75,11 +81,17 @@ public BuddyBalanceSubsystem() {
     tunerNumLeftI = new TunableNumber("Buddy Balance Left Motor I", kDefaultMotorI, leftPIDController::setI);
     tunerNumLeftD = new TunableNumber("Buddy Balance Left Motor D", kDefaultMotorD, leftPIDController::setD);
 
-    refPointBalancedTuner = new TunableNumber("Ref Point Balanced", 15, (a) -> {kBalancedPosition = a;});
-    refPointDeployedTuner = new TunableNumber("Ref Point Deployed", 0, (a) -> {kDeployedPosition = a;});
-    refPointServoTuner = new TunableNumber("Ref Point Servo", 1, (a) -> {kServoDeployedPos = a;});
+    refPointBalancedTuner = new TunableNumber("Ref Point Balanced", 0, (a) -> {kRetrievedAngle = a;});
+    refPointDeployedTuner = new TunableNumber("Ref Point Deployed", 0, (a) -> {kDeployedAngle = a;});
+    refPointServoTuner = new TunableNumber("Ref Point Servo", 0, (a) -> {kServoDeployedPos = a;});
+  }
+
+  public double getRightAngle() {
+    return Math.round(AbsoluteEncoder.getPositionRadians(rightEncoder));
```

Comment:
This is going to round to the nearest radian, which are large units. i.e. 1 radian is 57 degrees. So it will be very inaccurate